### PR TITLE
add documentation for POST /users/status/ids for apiv4

### PR DESCRIPTION
**Pull request by cpanato**
implementation pr: https://github.com/mattermost/platform/pull/5894

### CODE REVIEW

File: v4/source/status.yaml [hunk 1]
```diff
@@ -16,3 +16,24 @@
           $ref: '#/responses/BadRequest'
         '401':
           $ref: '#/responses/Unauthorized'
+
+  /users/status/ids:
+    get:
+      tags:
+        - status
+      summary: Get users status
```
Comment (jwilander) [inline v4/source/status.yaml]: Can we change this to `Get user statuses by id`

File: v4/source/status.yaml [hunk 2]
```diff
@@ -16,3 +16,24 @@
...
+      description: |
+        Get a list of users status by id from the server.
```
Comment (jwilander) [inline v4/source/status.yaml]: And this to `Get a list of user statuses by id from the server.`

File: v4/source/status.yaml [hunk 3]
```diff
@@ -16,3 +16,24 @@
...
+        ##### Permissions
+        Must be authenticated.
+      responses:
+        '200':
+          description: Users status retrieval successful
```
Comment (jwilander) [inline v4/source/status.yaml]: And this `User statuses retrieval successful`

File: v4/source/status.yaml (hunk 1 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

File: v4/source/status.yaml (hunk 2 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

File: v4/source/status.yaml (hunk 3 above)
Comment (cpanato) [inline v4/source/status.yaml]: done

**Pull request merged**